### ISSUE TITLE
Android:- Class not found World, Body When calling world.createBody

### ISSUE THREAD

**Issue by androidflip** [finnstr/gdx-liquidfun-extension#6]
HI,
I am using Libgdx.
I follow the same steps as given in https://github.com/finnstr/gdx-liquidfun-extension/wiki/Setup.
This is working fine in Desktop application but gives run time error that class not found for world, body .
I added the external jar as described in Setup wiki page.
Please help I am stuck.

**Comment by finnstr** [comment]
Hey,
you need to make sure you organised your order and export for android
correctly. If you don't export the right libraries the system won't find
the code and you get the error. Good luck.
Am 05.02.2016 08:30 schrieb "androidflip" <notifications@github.com>:

**Comment by androidflip** [comment]
Yes sir, I added the jars files and also selected order and export. But don't know why It shows such error.
Can you share 'HelloWorld' sample for the same, from where I can run LiquidFun for android using libgdx. 
It would be very helpful to us.
Thanks!

**Comment by progfiles** [comment]
same crash

**Comment by Tinlok** [comment]
same crash,please help me
